web-app/src/pages/Login/LoginService.js

```diff
@@ -44,7 +44,7 @@ export default class LoginService {
   }
 
   setUserEmail(email) {
-    localStorage.setItem('user_email', email);   
+    localStorage.setItem('user_email', email);
```

Comment:
This is something I added because we had no way of getting the user ID save for getting the profile with the API call that takes parameters 'firstname' and 'surname', which could only at the time be retrieved from the successful email. If we're changing login from JWT to base64 encode with user ID in payload then this method can go.

Reply:
Cool makes sense!